tests/codegen/issue_1226.jakt

```diff
@@ -0,0 +1,8 @@
+/// Expect:
+/// - output: "PASS\n"
+
+import issue_1226_sub
+
+function main() {
+   println("PASS")
```

Comment:
For the test, should this be calling the static functions, or do you need to not call the static function to cause the issue?

Reply:
Don't need to call the static function. The test will fail without the change.